### ISSUE TITLE
Ulnar-sided wrist pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10722]
**Preferred term label:**
Ulnar-sided wrist pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain at the ulnar fovea; DRUJ pain

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030836

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Triangular fibrocartilage complex
Distal radioulnar joint (DRUJ) arthritis
Pisotriquetral (PT) osteoarthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001200
- label: Ulnar wrist pain
- parents: [Wrist pain (HP:0030836)](https://hpo.jax.org/app/browse/term/HP:0030836)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the ulnar side of the wrist.
- comment: The ulnar side of the wrist is the side of the wrist closest to the little finger (fifth finger, pinky). Conditions characterized by this term include triangular fibrocartilage complex, distal radioulnar joint (DRUJ) arthritis, and pisotriquetral (PT) osteoarthritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10722
- synonyms:
